client/payment-methods/index.js

```diff
@@ -97,7 +101,7 @@ const UpeSetupBanner = () => {
 
 				<div className="payment-methods__express-checkouts-actions">
 					<span className="payment-methods__express-checkouts-get-started">
-						<Button isPrimary onClick={ handleEnableUpeClick }>
+						<Button isSecondary onClick={ handleEnableUpeClick }>
```

Comment:
`isPrimary` and `isSecondary` are both deprecated attributes from the `<Button>` component

Next time, could you try to migrate to the new suggested way (`variant="secondary"` in this case) when you encounter deprecated usages, please?